zulipterminal/helper.py

```diff
@@ -816,3 +817,61 @@ def open_media(controller: Any, tool: str, media_path: str) -> None:
 
     if error:
         controller.report_error(error)
+
+
+def user_friendly_time(time_stamp: str) -> str:  # message_info has a different format
```

Comment:
Instead of writing what the function does in the commit message, I think it would be more convenient to write it as comments within the function body, so that the next person working on it doesn't have to search for the respective commit(s).